oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -948,53 +979,10 @@ impl super::Node for WasmNode {
             self.config_name, self.entrypoint
         );
 
-        // Clone or copy all the captured values and move them into the closure, for simplicity.
```

Comment:
This is code from before... but is it possible to refactor the block of code above this line into a method, possibly called `enumerate_exports`, to make this `start` method read more similarly to the other `start` methods?

Reply:
Good call, I clarified the logic and added more logging since I was at it.